tests/forms/FormFieldTest.php

```diff
@@ -3,7 +3,52 @@
  * @package framework
  * @subpackage tests
  */
-class FormFieldTest extends SapphireTest implements TestOnly {
+class FormFieldTest extends SapphireTest {
+
+	public function testDefaultClasses() {
+		Config::nest();
+
+		Config::inst()->update('FormField', 'default_classes', array(
+			'class1',
+		));
+
+		$field = new FormField('MyField');
+
+		$this->assertContains('class1', $field->extraClass(), 'Default class not added to class list');
+
```

Comment:
The message in the final parameter is shown if the assertion fails.

So, in this example, we would state the message if the `extraClass` method didn't contain "class1" and therefore is a failure because the default class wasn't added.